viewer/models.py

```diff
@@ -275,10 +275,19 @@ class ComputedMolecule(models.Model):
     # comes from compound
     #original_smiles = models.CharField(max_length=255)
     # link to pdb file for prot structure
+    # instead link to a Protein
+    protein = models.ForeignKey(Protein, null=True)
```

Comment:
add `on_delete=models.CASCADE` - this is what is making the build fail on the PR